examples-c/README.md

```diff
@@ -2,4 +2,26 @@
 
 This folder holds tests (prefixed with `t_`) and examples for the C bindings.
 
-Headers are located in `include/` and the static and dynamic libraries in `lib/`
\ No newline at end of file
+Headers are located in `include/` and the static and dynamic libraries
+in `lib/`.
+
+To compile and run the example / test case, for example:
+
+```sh
+gcc ethereum_bls_signatures.c -o ethereum_bls_signatures -I../include -L../lib -lconstantine
```

Comment:
```suggestion
clang ethereum_bls_signatures.c -o ethereum_bls_signatures -I../include -L../lib -lconstantine
```

I'd rather use Clang everywhere as example.

GCC performance is just bad https://github.com/mratsim/constantine/issues/357